lib/datadog/appsec/contrib/rack/request_middleware.rb

```diff
@@ -13,13 +13,26 @@ module Datadog
   module AppSec
     module Contrib
       module Rack
+        # Create an array of lowercased headers
```

Comment:
Minor: I don't think this comment adds a lot, reminds me of this meme lol :wink: :wink: :wink: :wink::

![](https://res.cloudinary.com/practicaldev/image/fetch/s--aIBPDZ9g--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_800/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/fal0udi31n6r3pwg8d73.png)